administrator/components/com_admin/sql/updates/mysql/6.0.0-2025-10-11.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `#__history`
+    ADD COLUMN `is_current` TINYINT(1) NOT NULL DEFAULT 0;
```

Comment:
```suggestion
    ADD COLUMN `is_current` TINYINT NOT NULL DEFAULT 0 /** CAN FAIL **/;
```
1. Please add `/** CAN FAIL **/` installer hint so it does not cause a broken update in case if the column already exists for some reason.
2. Display widths for integers are not valid anymore on MySQL 8.